404.html

```diff
@@ -3,25 +3,97 @@
 permalink: /404.html
 ---
 <div class="container">
-    <!--Error Block-->
-    <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-            <div class="error-v1">
-                <span class="error-v1-title">404</span>
-                <span>Page not found!</span>
-                <p>The requested URL was not found. </p>
-            </div>
-        </div>
+  <div class="row">
+    <div class="col-md-8 col-md-offset-2">
+      <h1>Page not found</h1>
+
+      <p>Sorry, we could not find the page you were looking for. You can <a href="/">go to the landing page</a> or use the menu at the top of the page to navigate to a specific area of the site. </p>
+
+      <div id="search-suggestions" style="display:none;">
+        <b>Maybe those suggestions help?</b>
+        <ul id="search-suggestions-list">
+          <li><a href="/">blablablabla</a></li>
+          <li><a href="/">blablablabla</a></li>
+          <li><a href="/">blablablabla</a></li>
+        <ul>
+      </div>
+
     </div>
-    <!--End Error Block-->
+  </div>
 </div>
 
-<!-- send a 404 event to google analytics.
-	Source: https://gist.github.com/pwenzel/eb4f04552ef410f60406 -->
-<script>
-if(window.googleAnalyticsEvents){
-    window.googleAnalyticsEvents.functions.push( function(environment) {
-        environment.ga('send', 'event', 'error', '404', 'page: ' + document.location.pathname + document.location.search + ' ref: ' + document.referrer, {'nonInteraction': 1});
-    });
-}
-</script>
+<!-- send a 404 event to google analytics.-->
+  <script>
+    ga('send', 'event', 'error', '404', 'page: ' + document.location.pathname + document.location.search + ' ref: ' + document.referrer);
+  </script>
+
+  <script type="text/javascript">
```

Comment:
since there's no comment here, it looks like the comment above is still valid and this is part of the google analytics event. either move 25 down one line or (preferably) add another comment here